array-manipulation-2/src/drop.js

```diff
@@ -0,0 +1,16 @@
+/* exported drop */
+
+/*  results []
+    for item in array from index count to end
+      add item to results
+*/
+function drop(array, count) {
+  if (count > array.length) {
+    return array;
+  }
```

Comment:
If the count is larger than the array, shouldn't this return an empty array? Meaning all entries were dropped